### DISCUSSION TITLE
Attach a json file for a tenant

### DISCUSSION THREAD

**Discussion by Antony7310** [netbox-community/netbox#15399] [General]
Hello,

I would like to feed the netbox by uploading a document (json) for each of the tenants of my environment. How can I do this? I was thinking of using the /media/ directory but I can't download anything there? And obviously, there is no Tenant/File property... Any ideas?

**Comment by candlerb** [comment]
I'm not sure exactly what you're trying to achieve here (i.e. how you're intending to use this JSON data), but I observe that you can upload a JSON document as a [Config Context](https://docs.netbox.dev/en/stable/models/extras/configcontext/) (`/extras/config-contexts/add/`), and you can assign it to a specific tenant.

This has the side benefit that when you're generating a configuration for a device (or VM), the relevant config context for the device's tenant will be merged into `device.get_config_context()`
